tests/Util/yaml_fixtures/add_item_on_config_with_bind.test

```diff
@@ -0,0 +1,28 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    bind:
+      $httpClient: '@http.client'
+
+  app.service:
+    class: App\Services\Service
+    arguments:
+      - argument_1: 'something'
+        argument_2: 'something'
+===
+$data['app.service']['arguments']['argument_3'] = "something";
+===
+services:
+ _defaults:
+   autowire: true
+   autoconfigure: true
+   bind:
+     $httpClient: '@http.client'
```

Comment:
The indentation is slightly off here, which is making the test fail (it's making the expected not match)

Reply:
I miss this thank you